source/operators/dependentApiSimpleOperator/docker/dependentApiSimpleOperator.py

```diff
@@ -63,6 +65,50 @@ def safe_get(default_value, dictionary, *paths):
 def implementationReady(depapiBody):
     return safe_get(None, depapiBody, "status", "implementation", "ready")
 
+def get_depapi_spec(depapi_name, depapi_namespace):
+    api_instance = kubernetes.client.CustomObjectsApi()
+    try:
+        depapi = api_instance.get_namespaced_custom_object(
+            DEPAPI_GROUP, DEPAPI_VERSION, depapi_namespace, DEPAPI_PLURAL, depapi_name
+        )
+        return depapi["spec"]
+    except ApiException as e:
+        if e.status == HTTP_NOT_FOUND:
+            logger.error(
+                f"setDependentAPIStatus: dependentapi {depapi_namespace}:{depapi_name} not found"
+            )
+        else:
+            raise kopf.TemporaryError(
+                f"setDependentAPIStatus: Exception in get_namespaced_custom_object: {e.body}"
+            )           
+def get_expapi():
```

Comment:
This file was formatted using `black` with the default settings. 
I would suggest, to do this again.

Black can be installed using pip.
To format code, just change into the folder and execute

```
$ black .
reformatted ~/git/oda-canvas/source/operators/dependentApiSimpleOperator/docker/dependentApiSimpleOperator.py

All done! ✨ 🍰 ✨
1 file reformatted, 1 file left unchanged.
````

Reply:
As suggested, run the black on the dependent API operator
![image](https://github.com/tmforum-oda/oda-canvas/assets/29706990/45e9bec1-d152-4900-b9f1-75ba42ed3498)

![image](https://github.com/tmforum-oda/oda-canvas/assets/29706990/fb3fffb0-1def-446e-ac1c-d05e142e95a9)